examples/az.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# This file is part of pycloudlib. See LICENSE file for license information.
+"""Basic examples of various lifecycle with an Azure instance."""
+
+import logging
+
+import pycloudlib
+
+
+cloud_config = """#cloud-config
+runcmd:
+  - echo 'hello' > /home/ubuntu/example.txt
+"""
+
+
+def demo():
+    """Show example of using the Azure library.
+
+    Connects to Azure and launches released image. Then runs
+    through a number of examples.
+
+    PS: we assume in this example that you are logged into
+    you Azure account
+    """
+    client = pycloudlib.Azure('azure')
```

Comment:
probably worth calling out the param name for clarity
```suggestion
    client = pycloudlib.Azure(tag='demo-tag')
```

Reply:
Updated